src/Sanitizers.cpp

```diff
@@ -7,16 +7,7 @@ namespace Internal {
 
 namespace {
```

Comment:
It looks like this file is deleted from the Makefile/CMakeLists.txt, but it is not fully deleted here?